holoviews/core/spaces.py

```diff
@@ -478,6 +478,8 @@ def clone(self, callable=None, **overrides):
 
 
     def __call__(self, *args, **kwargs):
+        # Nothing to do for callbacks that accept no arguments
+        if not args and not kwargs: return self.callable()
```

Comment:
Just to be completely clear: There can be no stream *parameters* to worry about at this level. This becomes obvious in my comments below.